app/helpers/puppet_related_helper.rb

```diff
@@ -68,7 +68,7 @@ def interesting_puppetclasses(obj)
   end
 
   def extraction_warning
-    "being extracted to the foreman_puppet plugin and will be removed from core in 3.0"
+    _("being extracted to the foreman_puppet plugin and will be removed from core in 3.0")
```

Comment:
We can't use this to embed in another translated string. it won't work.

We would need other strings to be 
`_("Config Groups are %{common_extraction_warning}s") % { common_extraction_warning: extraction_warning }`

Reply:
But that leads to odd situations. In other languages the order of the sentence may be different. Perhaps it should be
```ruby
# singular
_("%s is being extracted to the foreman_puppet plugin and will be removed from core in 3.0") % item
# plural
_("%s are being extracted to the foreman_puppet plugin and will be removed from core in 3.0") % item
```

The only question I'd have is whether the items are always singular (in all translations) or plural.